index.dd

```diff
@@ -83,9 +83,9 @@ $(DIVID news,
 $(DIVC boxes,
     $(DIVC row,
         $(TOUR newspaper-o, News,
-            $(P The D foundation got accepted for the Google Summer of Code 2016
-                and $(LINK2 https://summerofcode.withgoogle.com/organizations/5078256051027968/, four projects)
-                will be pursued over the summer.
+            $(P $(LINK2 http://dconf.org/2016/index.html, DConf 2016) has concluded,
+                $(LINK2 https://vimeo.com/167235872, take a look),
+                and see you next year.
```

Comment:
- DConf 2016 was a big and bright week - we are looking forward to seeing you next year
- DConf 2016 was a big and bright week, watch the summary and see you next time.
- DConf 2016 was a full success, take an impression on hopefully see you next year
....